src/main/java/com/google/devtools/build/lib/cmdline/Label.java

```diff
@@ -438,10 +438,29 @@ public String getUnambiguousCanonicalForm() {
    * @param mainRepositoryMapping the {@link RepositoryMapping} of the main repository
    * @return analogous to {@link PackageIdentifier#getDisplayForm(RepositoryMapping)}
    */
-  public String getDisplayForm(RepositoryMapping mainRepositoryMapping) {
+  public String getDisplayForm(@Nullable RepositoryMapping mainRepositoryMapping) {
     return packageIdentifier.getDisplayForm(mainRepositoryMapping) + ":" + name;
   }
 
+  @StarlarkMethod(
+      name = "to_display_form",
```

Comment:
If we don't make it clear that this is a method rather than a field, I would find it pretty confusing to have next to e.g. `repo_name`. The `to` makes it so that users are very unlikely to do `print("My label: " + my_label.display_form)` and wonder why that fails.